### PR TITLE
Add pre commit

### PR DESCRIPTION
**Pull request by jgersak**

### CODE REVIEW

File: batch-processing/how_to_lulc/HowToLULC_batch.ipynb [hunk 1]
```diff
@@ -1338,11 +1338,11 @@
     "        :type bandname: str\n",
     "        \"\"\"\n",
     "        # Open with Rasterio\n",
-    "        with rasterio.Env(rasterio.session.AWSSession(AWS_session)) as env:\n",
-    "            s3_url = f\"s3://{bucket}/{request_id}/{folder}/{bandname}\"\n",
-    "            with rasterio.open(s3_url) as source:\n",
-    "                bbox = BBox(source.bounds, CRS(source.crs.to_epsg()))\n",
-    "                data = source.read()\n",
+    "        # with rasterio.Env(rasterio.session.AWSSession(AWS_session)) as env:\n",
```
Comment (batic) [inline batch-processing/how_to_lulc/HowToLULC_batch.ipynb:1341]: Something fishy is happening here; why did linting comment out this line?

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 2]
```diff
@@ -454,12 +441,12 @@
     "        tile_splitter = BBoxSplitter([aoi_shape], t_crs, (width_grid, heigth_grid))\n",
     "\n",
     "        tile_list = np.array(tile_splitter.get_bbox_list())\n",
-    "        info_list = np.array(tile_splitter.get_info_list())\n",
+    "        # info_list = np.array(tile_splitter.get_info_list())\n",
```
Comment (batic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:444]: Again, something fishy, this probably shouldn't be commented out.

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 3]
```diff
@@ -2565,9 +2535,9 @@
     "    parallel_batch_size = pred_p[\"single_batch_size\"] * pred_p[\"n_gpus\"]\n",
     "\n",
     "    # Load model parameters for printing\n",
-    "    with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
-    "        params_yaml = f_model_params.read()\n",
-    "        model_params = yaml.load(params_yaml)\n",
+    "    # with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
```
Comment (batic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2538]: Again.

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 4]
```diff
@@ -2790,9 +2760,9 @@
     "    )\n",
     "\n",
     "    all_preds = list()\n",
-    "    with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
-    "        params_yaml = f_model_params.read()\n",
-    "        model_params = yaml.load(params_yaml)\n",
+    "    # with open(op.join(ckpt_dir, pred_p[\"model_params_fname\"]), \"r\") as f_model_params:\n",
```
Comment (batic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2763]: And here as well.

File: poverty-detection-keras/Poverty_Detection_Workflow.ipynb [hunk 5]
```diff
@@ -2815,8 +2785,8 @@
     "        ###############################\n",
     "        # Predict on batches of images\n",
     "        ###############################\n",
-    "        y_preds = []\n",
-    "        pred_dict = dict()\n",
+    "        # y_preds = []\n",
```
Comment (batic) [inline poverty-detection-keras/Poverty_Detection_Workflow.ipynb:2788]: Here.

**Review by batic — COMMENTED** [review]
Looks good, thanks for adding this.

However, there are some lines that are now commented out and weren't before. Could you please check?

File: super-resolution-fastai/Train.ipynb
Comment (zigaLuksic) [inline super-resolution-fastai/Train.ipynb:1]: why was this removed?

**Review by zigaLuksic — CHANGES_REQUESTED** [review]
Prior to starting this we agreed that the pre-commit hooks would be formatters only.
If you want to enable flake8 as well, you need to do it in 2 steps (2 PRs).
1. enable formatters
2. enable flake8 and adjust code

Because both steps were combined we now have a HUGE pull request that spans 50 files. Noone can check that your adjustments in step 2. were safe.

Please close this PR and do it in two steps

**Pull request closed without merging**